### DISCUSSION TITLE
Hovering over a tooltip

### DISCUSSION THREAD

**Discussion by marcospassos** [ariakit/ariakit#1626] [Questions] [answered]
I'm building a popup that is only visible when hovering an element (anchor). It should allow you to hover inside the content of that popup to see additional information. 

I'm wondering what is the best approach: should I use a popover to manage the anchor in/out events, or use a tooltip and somehow keep it visible while hovering over it? I can't find any straightforward way to keep the tooltip opened while hovering over it.

Any tip would be very appreciated :)

**Comment by diegohaz** [comment, marked as answer]
You can use the Hovercard component.

**Reply by marcospassos** [reply]
This is exactly what I'm looking for! Thank you!
